packages/synapse-types/package.json

```diff
@@ -21,14 +20,12 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "build": "rimraf ./dist && tsup ./src/index.ts --format esm,cjs --dts",
-    "type-check": "tsc --noEmit",
+    "build": "tsc --build",
```

Comment:
Replaced tsup with tsc. Outside of the monorepo, this is used in SynapseWebClient e2e tests, where this seemed to be fine, so I think this is a safe change.